kura/org.eclipse.kura.wire.h2db.component.provider/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,7 @@ Bundle-Version: 2.3.0.qualifier
 Bundle-Vendor: Eclipse Kura
 Bundle-License: Eclipse Public License v2.0
 Bundle-Category: Kura Wires
-Import-Package: org.apache.logging.log4j;version="2.8.2",
+Import-Package: org.apache.logging.log4j;version="2.19.0",
```

Comment:
The considerations for the `org.slf4j` import should apply also for this package